### DISCUSSION TITLE
Publishing on Docker Hub

### DISCUSSION THREAD

**Discussion by benjaminrancourt** [umami-software/umami#915] [General]
Hi,

Thank you for your application, it is a simple and great alternative to Google Analytics. 😀

Is there any reason why you aren't publish images on Docker Hub?

Personnaly, I use the [docker notify](https://docker-notify.com/) service to be notified when a new image is released for the containers I am using and it doesn't support yet GitHub Packages...

If you want to support both, GitHub has created a documentation [here](https://docs.github.com/en/actions/publishing-packages/publishing-docker-images#publishing-images-to-docker-hub-and-github-packages) to publish on both platforms simultaneously. Let me know if I can implement it for you.

Psst. If you want to opt in your repository to the [Hacktoberfest](https://hacktoberfest.digitalocean.com/) event, you can [add the `Hacktoberfest` topic](https://hacktoberfest.digitalocean.com/resources/maintainers) to your repository. 🎃

**Comment by mikecao** [comment]
I don't use docker personally so that's probably why. I am open to a PR.

**Reply by sparanoid** [reply]
For the Docker Hub support. It needs to have a Docker user and token with write access like `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN` configured in secrets settings of this repo. After setting up the credentials I can open a PR for this feature.

**Comment by ajfriesen** [comment]
@ranb2002 
You can also use something like https://github.com/containrrr/watchtower to automatically update your containers.

You just need to be careful which tag you use and this will auto update your containers.

**Comment by eallion** [comment]
@mikecao 
ghcr.io (even gcr.io) is blocked in mainland China. We can only pull images from Docker Hub with a third-part Docker mirror. But those mirrors only work for the Docker Hub. (e.g. Alibaba Cloud)
Thanks!

**Reply by mikecao** [reply]
Sorry for the delay, finally got around to it, now on docker hub as `umamisoftware/umami`.

**Reply by eallion** [reply]
Thanks very much!
